### PR TITLE
feat: Onboard census bureau international dataset

### PR DESCRIPTION
**Pull request by nlarge-google**
## Checklist

Note: Delete items below that aren't applicable to your pull request.

- [x] Please merge this PR for me once it is approved.
- [x] If this PR adds or edits a dataset or pipeline, it was reviewed and approved by the Google Cloud Public Datasets team beforehand.
- [x] If this PR adds or edits a dataset or pipeline, I put all my code inside  `datasets/census_bureau_international` and nothing outside of that directory.
- [x] This PR is appropriately labeled.

### CODE REVIEW

**Review by adlersantos — CHANGES_REQUESTED** [review]
This PR uses too many images. Try to reduce the number of images, or better yet, use a single image for seemingly related pipelines.

File: datasets/census_bureau_international/_images/run_csv_transform_kub_age_specific_fertility_rates/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+numpy
+pandas
+google-cloud-storage
+gsutil
```
[the remark this answers is not in the document]
Comment (nlarge-google) [reply datasets/census_bureau_international/_images/run_csv_transform_kub_age_specific_fertility_rates/requirements.txt]: Done

File: datasets/census_bureau_international/_images/run_csv_transform_kub_country_names_area/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+numpy
+pandas
+google-cloud-storage
+gsutil
```
[the remark this answers is not in the document]
Comment (nlarge-google) [reply datasets/census_bureau_international/_images/run_csv_transform_kub_country_names_area/requirements.txt]: Done

File: datasets/census_bureau_international/_images/run_csv_transform_kub_country_names_area/csv_transform.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+
+def main(
+    source_url: str,
+    source_file: pathlib.Path,
+    target_file: pathlib.Path,
+    chunksize: str,
+    target_gcs_bucket: str,
+    target_gcs_path: str,
+) -> None:
```
[the remark this answers is not in the document]
Comment (nlarge-google) [reply datasets/census_bureau_international/_images/run_csv_transform_kub_country_names_area/csv_transform.py]: Done

**Comment by nlarge-google** [comment]
@adlersantos Please review these changes.  Thanks, Nick.

**Comment by nlarge-google** [comment]
@adlersantos I Added birth_death_growth_rates to the list of pipelines.  Please review.  Thanks!

**Pull request closed without merging**